assets/partners/common/int-1.Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM --platform=linux/amd64 python:3.9-slim
+ENV PYTHONUNBUFFERED=1
+
+WORKDIR /app
+
+COPY requirements.txt .
+RUN pip install -r requirements.txt
+RUN yes | great_expectations init
```

Comment:
Am I missing a way to run this with a -y flag?

Reply:
so that flag is part of our root CLI command - this should get it working `great_expectations -y init`

we should probably change this but this will do for now!

Reply:
Ohh... this failed
```
great_expectations init -y
```

but verified your suggestion works. I'll update

Reply:
fixed in dcf4eb088